src/Storage/Bucket.php

```diff
@@ -671,4 +739,28 @@ public function name()
     {
         return $this->identity['bucket'];
     }
+
+    /**
+     * Returns whether the bucket with the given file prefix is writable.
+     * Tries to create a temporary file as a resumable upload which will
+     * not be completed (and cleaned up by GCS).
+     *
+     * @param  string $file Optional file to try to write.
+     * @return boolean
+     */
+    public function isWritable($file = null)
```

Comment:
I didn't see an api that tells the client who you are (which would be needed for cross-referencing against the ACLs). Am I missing it?